src/foam/nanos/ruler/predicate/PropertyEQValue.js

```diff
@@ -40,11 +40,12 @@ foam.CLASS({
     {
       name: 'f',
       javaCode: `
-        if ( getIsNew() ) {
+        FObject old = (FObject) OLD_OBJ.f(obj);
+        if ( getIsNew() ||
+             old == null ) {
```

Comment:
when isNew=false, we want to compare the oldObj.prop to the value and if the oldObj is null should we just return false? @nanoMichal 

(similar thing for ropertyNEQValue.js and PropertyEQProperty.js)

Reply:
without the check for old == null we have NPE because the old tests are blindly executed.

Reply:
we can add the null check before the last return statement.
```
return old != null && EQ(old.getClassInfo().getAxiomByName ...
```